node/src/Worker.ts

```diff
@@ -545,15 +575,8 @@ export class Worker<
 		this.#closed = true;
 
 		// Kill the worker process.
-		if (this.#child) {
-			// Remove event listeners but leave a fake 'error' hander to avoid
-			// propagation.
-			this.#child.removeAllListeners('exit');
-			this.#child.removeAllListeners('error');
-			this.#child.on('error', () => {});
-			this.#child.kill('SIGTERM');
-			this.#child = undefined;
-		}
+		this.#child.unref();
```

Comment:
I've randomly added this unref() call 😌

Reply:
And I've removed now since it is useless since we DO HAVE a IPC:

https://nodejs.org/api/child_process.html#subprocessunref